lib/publishing.ts

```diff
@@ -249,10 +258,12 @@ export class AssetPublishing implements IPublishProgress {
   }
 
   public get percentComplete() {
-    if (this.totalOperations === 0) {
-      return 100;
-    }
-    return Math.floor((this.completedOperations / this.totalOperations) * 100);
+    const completionProgress =
+      this.totalOperations === 0
+        ? 100
+        : Math.floor((this.completedOperations / this.totalOperations) * 100);
+    globalOutputHandler.setCompletionProgress(completionProgress);
```

Comment:
Same problem as with setting the listener, breaks on multiple instances of `AssetPublishing`